### ISSUE TITLE
test code

### ISSUE THREAD

**Issue by soninaren** [soninaren/sample#1]

**Comment by soninaren** [comment]
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <script src="//ajax.aspnetcdn.com/ajax/jQuery/jquery-1.11.3.min.js"></script>
    <script type='text/javascript'>
        $(window).on('load', function () {
            cells = document.getElementsByTagName('td');

            for (var i = 0, len = cells.length; i < len; i++) {
                if (cells[i].innerHTML.toLocaleLowerCase() == 'passed') {
                    cells[i].style.backgroundColor = 'green';
                    cells[i].style.color = 'white';
                }
                if (cells[i].innerHTML.toLocaleLowerCase() == 'failed') {
                    cells[i].style.backgroundColor = 'red';
                    cells[i].style.color = 'white';
                }
				
				if (cells[i].innerHTML.toLocaleLowerCase() == 'skipped') {
                    cells[i].style.backgroundColor = 'orange';
                    cells[i].style.color = 'white';
                }
            }
        });

    </script>

    <center>
    <h1>Test Results<br></h1>
    </center>
    <style>
        TABLE {
            width: 1080px;
            margin-left: auto;
            margin-right: auto;
            border-width: 1px;
            border-style: solid;
            border-color: black;
            border-collapse: collapse;
        }

    .green {
        background-color: green;
    }

    .red {
        background-color: red;
    }

        tr:hover {
            background-color: lightBlue;
            color: white;
        }

        TH {
            color: white;
            border-width: 1px;
            padding: 3px;
            border-style: solid;
            border-color: black;
            background-color: #6495ED;
        }

        TD {            
            border-width: 1px;
            padding: 3px;
            border-style: solid;
            border-color: black;
        }

        TD {
            border-width: 1px;
            padding: 3px;
            border-style: solid;
            border-color: black;
        }

        td:first-child {
            padding: 6px;
	        white-space: nowrap;
	    }

		td:nth-child(2) {
    	    padding: 6px;
	    	white-space: nowrap;
		}
	
		td:nth-child(3) {        
            padding: 6px;
	    	text-align: center;	
	    	white-space: nowrap;
		}
    </style>    
</head><body>
<table>
<colgroup><col/><col/><col/><col/></colgroup>
<tr><th>Template</th><th>Action</th><th>Status</th><th>Log</th></tr>
<tr><td>BlobTrigger-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>BlobTrigger-CSharp</td><td>Blob Trigger executed</td><td>passed</td><td>Execute Trigger</td></tr>
<tr><td>BlobTrigger-CSharp</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:49:05.556 Function started (Id=ff45b232-5d5e-4e6f-a9d2-130a716c46c2)
2016-04-07T20:49:05.556 C# Blob trigger function processed: My sample data
2016-04-07T20:49:05.556 Function completed (Success, Id=ff45b232-5d5e-4e6f-a9d2-130a716c46c2)
</td></tr>
<tr><td>BlobTrigger-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>BlobTrigger-NodeJS</td><td>Blob Trigger executed</td><td>passed</td><td>Execute Trigger</td></tr>
<tr><td>BlobTrigger-NodeJS</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:49:21.657 Function started (Id=e65d42a9-a121-4bda-85bb-3e76aa532e40)
2016-04-07T20:49:22.314 Node.js blob trigger function processed blob: undefined
2016-04-07T20:49:22.314 Function completed (Success, Id=e65d42a9-a121-4bda-85bb-3e76aa532e40)
2016-04-07T20:50:01.829 Function started (Id=d8e7aeea-6c09-467b-bf1b-19007236c3ed)
2016-04-07T20:50:01.860 Node.js blob trigger function processed blob: foobar123
2016-04-07T20:50:01.860 Function completed (Success, Id=d8e7aeea-6c09-467b-bf1b-19007236c3ed)
</td></tr>
<tr><td>GenericWebHook-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>GenericWebHook-CSharp</td><td>Execute Trigger</td><td>passed</td><td>found matching value:{
    &quot;greeting&quot;:  &quot;Hello Azure Functions!&quot;
}</td></tr>
<tr><td>GenericWebHook-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>GenericWebHook-NodeJS</td><td>Execute Trigger</td><td>passed</td><td>found matching value:{
    &quot;greeting&quot;:  &quot;Hello Azure Functions!&quot;
}</td></tr>
<tr><td>GitHubWebHook-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>GitHubWebHook-CSharp</td><td>Execute Trigger</td><td>passed</td><td>found matching value:&quot;From Github:This is a comment on a GitHub issue&quot;</td></tr>
<tr><td>GitHubWebHook-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>GitHubWebHook-NodeJS</td><td>Execute Trigger</td><td>passed</td><td>found matching value:&quot;New GitHub comment: This is a comment on a GitHub issue&quot;</td></tr>
<tr><td>HttpTrigger-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>HttpTrigger-CSharp</td><td>Execute Trigger</td><td>passed</td><td>found matching value:Hello Azure</td></tr>
<tr><td>HttpTrigger-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>HttpTrigger-NodeJS</td><td>Execute Trigger</td><td>passed</td><td>found matching value:Hello Azure</td></tr>
<tr><td>QueueTrigger-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>QueueTrigger-CSharp</td><td>Queue Trigger executed</td><td>passed</td><td>Execute Trigger</td></tr>
<tr><td>QueueTrigger-CSharp</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:51:08.624 Function started (Id=84324485-601b-4b17-ac2b-3b4616214d89)
2016-04-07T20:51:08.624 C# Queue trigger function processed: sample queue data
2016-04-07T20:51:08.624 Function completed (Success, Id=84324485-601b-4b17-ac2b-3b4616214d89)
</td></tr>
<tr><td>QueueTrigger-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>QueueTrigger-NodeJS</td><td>Queue Trigger executed</td><td>passed</td><td>Execute Trigger</td></tr>
<tr><td>QueueTrigger-NodeJS</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:51:17.907 Function started (Id=e61eae81-d5be-4fcc-a593-2462a56884ca)
2016-04-07T20:51:17.940 Node.js queue trigger function processed work item sample queue data
2016-04-07T20:51:17.940 Function completed (Success, Id=e61eae81-d5be-4fcc-a593-2462a56884ca)
</td></tr>
<tr><td>TimerTrigger-CSharp</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>TimerTrigger-CSharp</td><td>Execute Trigger</td><td>passed</td><td>Timer trigger will auto execute</td></tr>
<tr><td>TimerTrigger-CSharp</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:51:27.118 Function started (Id=23572ff5-66c6-4b33-84a9-977e75c3e878)
2016-04-07T20:51:27.118 C# Timer trigger function executed at: 4/7/2016 8:51:27 PM
2016-04-07T20:51:27.118 Function completed (Success, Id=23572ff5-66c6-4b33-84a9-977e75c3e878)
</td></tr>
<tr><td>TimerTrigger-NodeJS</td><td>Deploy Template</td><td>passed</td><td>Deployment Successful</td></tr>
<tr><td>TimerTrigger-NodeJS</td><td>Execute Trigger</td><td>passed</td><td>Timer trigger will auto execute</td></tr>
<tr><td>TimerTrigger-NodeJS</td><td>Check Logs</td><td>passed</td><td>found matching value:2016-04-07T20:51:36.043 Function started (Id=b51a8c1d-90b1-4704-b2ae-c68d241ec373)
2016-04-07T20:51:36.075 Node.js is running late!
2016-04-07T20:51:36.075 Node.js timer trigger function ran! 2016-04-07T20:51:36.075Z
2016-04-07T20:51:36.075 Function completed (Success, Id=b51a8c1d-90b1-4704-b2ae-c68d241ec373)
</td></tr>
</table>
</body></html>
